enzyme/Enzyme/CApi.cpp

```diff
@@ -646,8 +646,7 @@ LLVMValueRef EnzymeGradientUtilsCallWithInvertedBundles(
   auto callval = unwrap(func);
 
 #if LLVM_VERSION_MAJOR > 7
-  auto res = BR.CreateCall(
-      cast<FunctionType>(callval->getType()->getPointerElementType()), callval,
+  auto res = BR.CreateCall(isa<CallBase>(callval)->getFunctionType(), callval,
```

Comment:
callval is a literal value being called, not a callbase.

Reply:
Oh, I see. This will require an API change to fix then.